index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<!--[if lte IE 6]><html class="preIE7 preIE8 preIE9"><![endif]-->
+<!--[if IE 7]><html class="preIE8 preIE9"><![endif]-->
+<!--[if IE 8]><html class="preIE9"><![endif]-->
+<!--[if gte IE 9]><!--><html><!--<![endif]-->
+  <head>
+    <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>My ReDi Progject</title>
+  <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+  <link rel="stylesheet" href="stylesheet.css" type="text/css">
+  </head>
+  <body>
+      
+      <header>
+          <nav> 
+                <a href="">HOME</a>
+                <a href="">ABOUT ME</a>  
+                <a href="index.html" class="current">BASE PAGE</a>
+          </nav>
+      </header>
+      <div class="main">        
+          <section class="post">
+              <h2>Traveling in Bavaria</h2>
+                 <div>
+              <img class="postpic" src="travel.jpg" >
```

Comment:
One of hardest parts of programming in naming things. `post` is a great name. The sub elements of your post have class names that are too generic. This does not matter now because your project is very small. In the future when you work on big projects being familiar with a naming convention will come in handy. A lot of frontend teams agree to a pattern for naming there classes in this case. I've found default standard to be [BEM](http://getbem.com/naming/). 

In this case for your `post` is your block and things like `pic` are the element so classes like

- `post_pic`
- `post_description`
- `post_link`

then for example if you had a red link for some reason
- `post_link--red`

would work quite well